packages/ketcher-react/src/script/ui/state/shared.js

```diff
@@ -75,6 +75,10 @@ export function load(struct, options) {
     return parseStruct(struct, server, options)
       .then(
         struct => {
+          if (!struct) {
```

Comment:
It looks like workaround. I guess 'then' expression  should be only executed if struct is not empty. Could you  find the reason of this case?